sklearn/discriminant_analysis.py

```diff
@@ -623,11 +623,15 @@ class QuadraticDiscriminantAnalysis(BaseEstimator, ClassifierMixin):
         Discriminant Analysis
     """
 
-    def __init__(self, priors=None, reg_param=0., store_covariances=False,
-                 tol=1.0e-4):
+    def __init__(self, priors=None, reg_param=0.,store_covariance=False, tol=1.0e-4,store_covariances=None):
+        if store_covariances is not None:
+        	warnings.warn("'store_covariances' was renamed to 'store_covariance' in version 0.17 and "
+                      "will be removed in later versions.", DeprecationWarning)
+        	self.store_covariance = store_covariances
```

Comment:
But in the docs it was mentioned that if we want to deprecate a parametre, if that is not none, then we need to assign the value of the old parametre to the new parametre.